src/photosui.cs

```diff
@@ -24,8 +24,8 @@ namespace PhotosUI {
 	[NoTV]
 	[iOS (8, 0)]
 	[Mac (10, 13)]
+	[MacCatalyst (14,0)]
 	[Protocol]
-	[Unavailable (PlatformName.MacCatalyst)]
 	[Advice ("This API is not available when using UIKit on macOS.")]
```

Comment:
You need to remove this advice too if you remove `[Unavailable (PlatformName.MacCatalyst)]`

```suggestion
```